src/zenml/models/v2/core/pipeline_run.py

```diff
@@ -235,6 +235,9 @@ class PipelineRunResponseMetadata(WorkspaceScopedResponseMetadata):
         default=None,
         description="Template used for the pipeline run.",
     )
+    is_templatable: bool = Field(
```

Comment:
This is probably intended, but I just wanted draw attention to the fact that this is a required field without a default. You might have to adjust some of the fixtures for our tests.

Reply:
Yep that's intended, let's see if our tests fail

Reply:
There is one [here](https://github.com/zenml-io/zenml/blob/be215d157c27be99206429d3ca703c94ff46a542/tests/unit/conftest.py#L447).